src/expressive.annotations.validate.js

```diff
@@ -524,13 +524,12 @@ var
                             namespacedEvents.push(typeHelper.string.format('{0}.expressive.annotations', event));
                         }
                     });
-                    $(form).find('input, select, textarea').on(namespacedEvents.join(' '), function(event) {
+                    // attach handlers to all inputs that do not have 'ea-bound' class (unless force == true)
+                    $(form).find('input, select, textarea').not(function(idx,el){ return $(el).hasClass('ea-bound') || force;}).on(namespacedEvents.join(' '), function () {

```

Comment:
Unfortunately this line is not going to work as you are expecting it to. You're appending the 'ea-bound' class to the input field when an event is already invoked. Till that time, it could be bound dozens of times, so it will be invoked dozens of times. If you'll check that logic on provided mvc sample project, you'll notice how sluggish the application has became, because of how many times this event is invoked.